content/new-in-22.md

```diff
@@ -199,40 +189,51 @@ Provide ways to help users navigate, find content, and determine where they are.
 </blockquote>
 <p><a href="https://www.w3.org/WAI/WCAG22/Understanding/focus-not-obscured-minimum">Understanding Focus Not Obscured (Minimum)</a></p>
 
-### 2.4.12 Focus Not Obscured (Enhanced) (AAA)
-<p><em>(Persona, problem, and works well<!--, and "in brief" --> same as 2.4.12 above.)</em></p>
+### [EOWG draft] 2.4.12 Focus Not Obscured (Enhanced) (AAA)
+
+<p>In brief:</p>
+<ul>
+      <li><p><span class="whatwhy">What to do: </span>Ensure that when items have focus, they are fully visible.</p></li>
+      <li><p><span class="whatwhy">Why do it: </span>Some people cannot use a mouse and rely on seeing what has keyboard focus.</p> </li>
+</ul>
+<p><em>(Persona, problem, and works well same as 2.4.12 above.)</em></p>
 <p class="sclabel">WCAG:</p>
 <blockquote class="sc">
   <p>When a <a href="https://www.w3.org/TR/WCAG22/#dfn-user-interface-components">user interface component</a> receives keyboard focus, no part of the component is hidden by author-created content.</p>
 </blockquote>
 <p><a href="https://www.w3.org/WAI/WCAG22/Understanding/focus-not-obscured-enhanced">Understanding Focus Not Obscured (Enhanced)</a></p>
 
-### 2.4.13 Focus Appearance (AAA)
-<p class="persona"><a href="https://www.w3.org/WAI/people-use-web/user-stories/#reporter">Reporter</a> with repetitive stress injury who doesn't use a mouse:<br>
-  and <a href="https://www.w3.org/WAI/people-use-web/user-stories/#retiree">Retiree</a> with low contrast sensitivity:</p>
-<div class="quotes">
-  <ul>
-    <li>
-      <p><span class="issue">Problem:</span><span><q>I can't tell where the keyboard focus is as I move around a web page or app.</q></span></p>
-    </li>
-    <li>
-      <p><span class="issue">Works well:</span><span><q>I can see where the keyboard focus is as I move around a web page or app.</q></span></p>
-    </li>
-  </ul>
-</div>
-<!--
+### [AG version] 2.4.13 Focus Appearance (AAA)
 <section class="brief">
   <p>In brief:</p>
   <dl>
     <dt>Objective</dt>
     <dd>Make it easier to see what has keyboard focus</dd>
     <dt>Author task</dt>
-    <dd>The focus indicator contrasts 3:1 with both the component’s unfocused state and its surroundings</dd>
+    <dd>The focus indicator color contrasts 3:1 with both the component’s unfocused state and its surroundings</dd>
     <dt>Key beneficiaries</dt>
     <dd>Sighted users reliant on keyboard interaction</dd>
   </dl>
 </section>
--->
+
+### [EOWG draft] 2.4.13 Focus Appearance (AAA)
+<p>In brief:</p>
+<ul>
+      <li><p><span class="whatwhy">What to do: </span>Ensure that the focus indicator contrasts 3:1 with the component’s unfocused state and with its surroundings.</p></li>
```

Comment:
From [Brian](https://www.w3.org/2002/09/wbs/35532/wcag2_2_in_brief/results):
I don't think we should have technical items in these descriptions, such as the 3:1 ratio. The description also seems to miss the need for the focus indication to be bolder (2px width). Can we have a less technical and more encompassing statement? 
"Ensure that the focus indicator is bold and has a high contrast with the component’s unfocused state and with its surroundings." or similar

Reply:
Well, we want it to be actionable, and this is the Understanding SC doc, so I think good to have technical detail where relevant and not confusing. I wonder if the idea of contrast ratios is well enough known among accessibility folks that it's OK?
I don't think of 2px width as "bold".
3:1 is not "high contrast" :-)

I added 2px to see what folks think...